src/model/window-guardian.ts

```diff
@@ -24,6 +24,9 @@ export interface WindowGuardianConfig {
     };
     switches: { [key: string]: boolean };
     tests?: { [key: string]: string };
+    ophan?: {
+        pageViewId: string;
+    };
```

Comment:
Is `ophan.pageViewId` the same as `CAPI.pageId`?

Reply:
I don't think so. It's generated in the HTML document directly when the app is rendered on the server. I don't think we have anything in place to use this on the server yet.
https://github.com/guardian/dotcom-rendering/blob/b5e01f6d2ec98122a0ca443793bde4aa145ae6d3/src/web/server/htmlTemplate.ts#L117